com/tpntpupd.m

```diff
@@ -165,14 +172,18 @@
 	if operation=5 do noop
 	quit
 
+tpnomultigldcheck(str)
+	if tpnomultigld set str="set $zgbldir=tpgld "_str
```

Comment:
Could be written as below to avoid the if logic (generates better code):

set:tpnomultigld str="..."...

No need to change this one or the several others in here but perhaps keep in mind for next time.

Reply:
Addressed in https://github.com/YottaDB/YottaDBtest/pull/359